roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,137 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DockerImageAvailability(OpenShiftCheck):
+    """Check that required Docker images are available.
+
+    This check attempts to pull required docker images and fail if
+    any image is missing or an error occurs during pull.
+    """
+
+    name = "docker_image_availability"
+    tags = ["preflight"]
+
+    skopeo_image = "juanvallejo/skopeo:latest"
```

Comment:
I did not see [a flag for force pulling](https://docs.docker.com/engine/reference/commandline/pull/#concurrent-downloads), but I updated the `update_skopeo_image` method to execute `docker pull` on the skopeo image on each run of the check